tools/throughput_linear_attention_jax.py

```
@@ -69,78 +68,83 @@ def _get_peak_bf16_flops() -> float | None:
-for kernel_backend, row_header in kernels:
-    mfu_row = [row_header]
-    bw_row = [row_header]
+def _measure(kernel_backend: KernelBackend, jax_dtype, run_forward: bool) -> float:
+    """seconds per iteration for one (kernel, dtype, pass) combination"""
+
+    key_q, key_k, key_v, key_dy, key_dht = jax.random.split(jax.random.PRNGKey(0), 5)
+    q = jax.random.normal(key_q, (B, S, N, K), dtype=jnp.float32).astype(jax_dtype)
+    k = jax.random.normal(key_k, (B, S, N, K), dtype=jnp.float32).astype(jax_dtype)
+    v = jax.random.normal(key_v, (B, S, N, V), dtype=jnp.float32).astype(jax_dtype)
+
+    fn = lambda q, k, v: linear_attention_jax(q, k, v, None, BLOCK_SIZE_S=BLOCK_SIZE_S, kernel_backend=kernel_backend)
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `fn` as a `def`

(E731)